scripts/build

```diff
@@ -3,7 +3,7 @@ set -e
 
 if ! command -v dotnet-format; then
   echo "Installing dotnet formatter..."
-  dotnet tool install -g dotnet-format
+  dotnet tool install -g dotnet-format --version 4.0.0
```

Comment:
`v3-v2019-10-10` version of https://github.com/recurly/recurly-client-dotnet/pull/612/files#r579476890